app/src/main/java/aloksharma/ufl/edu/stash/ServerAccess.java

```diff
@@ -106,10 +124,104 @@ private Double getBankBalance(String username, String password, BankName
         postArgs.add(new BasicNameValuePair("username", username));
         postArgs.add(new BasicNameValuePair("password", password));
         postArgs.add(new BasicNameValuePair("type", bankName.toString()));
-
         return plaidPostRequest(plaidUrl, postArgs);
     }
 
+   /******************Add Stash Functionality******************************************/
```

Comment:
To describe a function, follow the javadoc convention, which is a standard. For reference see [here](https://github.com/Alok-Sharma/Stash/blob/master/app/src/main/java/aloksharma/ufl/edu/stash/ServerAccess.java#L245)
Basically, a slash followed by two asterix.